packages/main/src/themes/sap_belize/Slider-parameters.css

```diff
@@ -0,0 +1,18 @@
+@import "../base/Slider-parameters.css";
+@import "../../../../theme-base/src/themes/sap_belize/base-parameters.less";
```

Comment:
@import "../../../../theme-base/src/themes/sap_belize/base-parameters.less";
 should not be needed

Reply:
Done